packages/configuration-web-recommended/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "dependencies": {
+    "@sonarwhal/formatter-codeframe": "^1.0.0",
+    "@sonarwhal/formatter-stylish": "^1.0.0",
+    "@sonarwhal/formatter-summary": "^1.0.0",
+    "@sonarwhal/rule-apple-touch-icons": "^1.0.0",
+    "@sonarwhal/rule-axe": "^1.0.0",
+    "@sonarwhal/rule-content-type": "^1.0.0",
+    "@sonarwhal/rule-disown-opener": "^1.0.0",
+    "@sonarwhal/rule-highest-available-document-mode": "^1.0.0",
+    "@sonarwhal/rule-html-checker": "^1.0.0",
+    "@sonarwhal/rule-http-cache": "^1.0.0",
+    "@sonarwhal/rule-http-compression": "^1.0.0",
+    "@sonarwhal/rule-manifest-app-name": "^1.0.0",
+    "@sonarwhal/rule-manifest-exists": "^1.0.0",
+    "@sonarwhal/rule-manifest-file-extension": "^1.0.0",
+    "@sonarwhal/rule-manifest-is-valid": "^1.0.0",
+    "@sonarwhal/rule-meta-charset-utf-8": "^1.0.0",
+    "@sonarwhal/rule-meta-viewport": "^1.0.0",
+    "@sonarwhal/rule-no-disallowed-headers": "^1.0.0",
+    "@sonarwhal/rule-no-friendly-error-pages": "^1.0.0",
+    "@sonarwhal/rule-no-html-only-headers": "^1.0.0",
+    "@sonarwhal/rule-no-http-redirects": "^1.0.0",
+    "@sonarwhal/rule-no-protocol-relative-urls": "^1.0.0",
+    "@sonarwhal/rule-no-vulnerable-javascript-libraries": "^1.0.1",
+    "@sonarwhal/rule-strict-transport-security": "^1.0.0",
+    "@sonarwhal/rule-validate-set-cookie-header": "^1.0.0",
+    "@sonarwhal/rule-x-content-type-options": "^1.0.0"
+  },
+  "description": "sonarwhal's recommended rules configuration for live websites",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "index.json"
+  ],
+  "keywords": [
+    "configuration",
+    "recommended",
```

Comment:
Do we need these? Otherwise I think we should remove it, it's to generic.

Reply:
Maybe remove `recommended` and `configuration` but I'd others like `best practices`, `security`, etc.Copy paste the ones from `sonarwhal`?

Reply:
Yeap, that's what I was referring to. "thouse" the above my comment ones.